db/migrate/20230614110657_create_like.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] In the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png), the many `Likes` belong to A `Post`. To illustrate that, you need to add a `post` ref to the `Likes` table. The `post` ref will help to map all Likes created to a particular post in the database. Kindly implement this. 👍 💻